### PR TITLE
Add instructions for starting Mongo before starting the app.

### PR DESCRIPTION
**Pull request by sarahcstringer**
The instructions do not mention starting MongoDB after installing it and before running the app; if the MongoDB server isn't running, the app will fail to start.

I also linked directly to the MongoDB community edition in the prerequisite step.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Implementations in other languages:
 ### Requirements
 
 - [Nodejs](https://nodejs.org/) v10 or v12
-- [Mongo](https://docs.mongodb.com/manual/installation/)
+- [Mongo](https://docs.mongodb.com/manual/administration/install-community/)
```
Comment (stern-shawn) [inline README.md:28]: 🙌

File: README.md [hunk 2]
```diff
@@ -74,6 +74,11 @@ Once you have created your TwiML app, [configure your Twilio phone number](https
 
 4. Run the application.
 
+   Make sure MongoDB is running before starting the application. See how to start the MongoDB service on
+   [Windows](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/#start-mongodb-community-edition-as-a-windows-service), [MacOS](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/#run-mongodb-community-edition), or [Linux](https://docs.mongodb.com/manual/administration/install-on-linux/) (choose your Linux distribution and then see "Run MongoDB Community Edition" in the installation instructions).
```
Comment (stern-shawn) [inline README.md:78]: Since this is necessary before running the app, would it be clearer to make this its own numbered step? Just an idea!

**Review by stern-shawn — APPROVED** [review]
Looks great + one idea!

File: README.md (hunk 2 above)
Comment (sarahcstringer) [reply README.md:78]: Makes sense! I'll split it out.

**Pull request merged**